Strimzi installation on Kubernetes (#1769)
ManageIQ/manageiq-documentation: installing_on_kubernetes/_topics/prepare_kubernetes_namespace.md

## Preparing the Kubernetes namespace
Prerequisite:

> If running on plain Kubernetes, manual installation of Kafka is required. ManageIQ uses [Strimzi](https://strimzi.io/) and installation steps can be found [here](https://strimzi.io/docs/operators/latest/deploying#deploy-tasks_str). Only the Strimzi cluster operator needs to be deployed, the ManageIQ operator will then make the necessary requests for the resources needed from Strimzi. However if you are using Openshift, the ManageIQ operator will install Strimzi for you and no action is required.

Reviewer: ```suggestion
If you are using OpenShift, our operator will leverage the OpenShift catalog to install Strimzi for you automatically.
If not running on OpenShift, manual installation of Strimzi Kafka is required. [Strimzi](https://strimzi.io/) installation steps can be found [here](https://strimzi.io/docs/operators/latest/deploying#deploy-tasks_str). Only the Strimzi cluster operator needs to be deployed, our operator will then make the necessary requests for the resources needed from Strimzi.
```

Reviewer: If the product name is necessary, please use `{{ site.data.product.title_short }}` so that the documentation can be productized.

Author: This doc is not used downstream, we confirmed with the docs team

   $ oc get crds | grep {{ site.data.product.operator_custom_resource_definition_name_plural }}.{{ site.data.product.operator_namespace }}
   $ oc create -f config/crd/bases/{{ site.data.product.operator_namespace }}_{{ site.data.product.operator_custom_resource_definition_name_plural }}.yaml
   ```

>    Prerequisite:

Reviewer: I think this should be just below `## Preparing the Kubernetes namespace` since everything else in this doc is ours.  This is the only external dependency,

Author: thanks, moved it to the top

   $ oc create -f config/rbac/service_account.yaml
   ```

4. Install Strimzi Kafka

> {% include_relative install_kafka.md %}

Reviewer 2: This is a little weird, but I'm not sure how to do it otherwise.  The reason it's weird is because the header title will appear under the number, and might break the numbered bulleting.  We'd have to see what a rendered page looks like.

Additionally, I'm not sure if we use this downstream, but if so, it needs to be productized.

   $ oc create -f config/rbac/service_account.yaml
   ```

> {% include_relative install_kafka.md %}

Reviewer: Should this be a prerequisite instead?
Maybe between lines 1&2 directly in this doc unless we're going to use it somewhere else.
